components/CareCharges/EditElementModal/EditElementDatePickers.js

```diff
@@ -0,0 +1,134 @@
+import DatePick from '../../DatePick';
+import RadioButton from '../../RadioButton';
+import React, { useState } from 'react';
+import { formatDate, incrementDate } from '../../../service/helpers';
```

Comment:
formatDate can be replaced with date-fns format... e.g. format(new Date(value), 'd/MM/Y')